src/layouts/SiteLaunchPad/components/SiteLaunchFinalState.tsx

```diff
@@ -72,13 +76,51 @@ const SiteLaunchFailureState = (): JSX.Element => {
   )
 }
 
+const SiteLaunchInProgressState = (): JSX.Element => {
+  return (
+    <>
+      <Text
+        textStyle="h3"
+        textColor="base.content.strong"
+        mt="3rem"
+        mb="1.5rem"
+      >
+        We&apos;re keeping an eye out... Be right back!
+      </Text>
+      <SiteLaunchPendingImage />
+      <Text
+        textStyle="body-1"
+        textColor="base.content.default"
+        mt="1.5rem"
+        textAlign="center"
+      >
+        Come back to this space to track your site status - you may see an error
+        when visiting your domain at this moment. Leave this window open or exit
```

Comment:
just curious leaving the window open - does it dynamically update the status of site launch?

Reply:
Yes, because we do a five min refetch behaviour in the background. This is also tested in the other pr - where I manually mimic the invalidation of the key + force a refetch, the screen gets updated. 

Design wanted to work on animation for this screen to signify to the user that this is a dynamic process, but this is a out of scope for now.